app/controllers/concerns/jsonapi_serializer_concerns.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+SERIALIZEABLE_AR_MODELS = %i[
+  Artist
+  ArtPiece
+  ArtPieceTag
+  Email
+  Medium
+  Studio
+].freeze
+
+module JsonapiSerializerConcerns
+  extend ActiveSupport::Concern
+
+  def jsonapi_class
+    super.merge(mau_serializer_lut)
+  end
+
+  def mau_serializer_lut
+    unless @config
+      @config = _generate_serializer_lut
+      @config.freeze
+    end
+    @config
+  end
+
+  def _generate_serializer_lut
+    SERIALIZEABLE_AR_MODELS.index_with { |val| "#{val}Serializer".constantize }.tap do |config|
+      config[:IndependentStudio] = StudioSerializer
+      config[:GenericEvent] = ApplicationEventSerializer
```

Comment:
I think these last 3 are unused because the `application_event` call does *not* go through the api/v2 services so it's not `jsonapi` but if this goes, we can move the other services over.